src/hooks/usePlayList.js

```diff
@@ -1,87 +1,64 @@
+// @ts-check
 import { useEffect, useState } from 'react'
 import getPlaylistData from '../utils/getPlaylistData'
+import getWrongAnswers from '../utils/getWrongAnswers'
 import shuffle from '../utils/shuffleArray'
 
 export default function usePlayList(playlist) {
-  const newPlaylist = shuffle(playlist)
+  const newPlaylist = playlist
```

Comment:
Why this rename?

Reply:
Right after i get rid of the shuffle i don't even need the const at all.